docs/source/receptorctl/receptorctl_ping.rst

```diff
@@ -0,0 +1,28 @@
+----
+ping
+----
+
+.. contents::
+   :local:
+
+``receptorctl ping`` Pings a receptornode.
```

Comment:
```suggestion
``receptorctl ping`` tests the network reachability of Receptor nodes.
```

Reply:
Not sure if s/reachability/connectivity is better here or not. I guess there is some nuance there. "reachability" implies that you can verify the target node is up and active whereas "connectivity" implies the target node is available to receive inbound requests.